org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/terminologies/utilities/ValueSetProcessBase.java

```diff
@@ -8,7 +8,7 @@
 import org.hl7.fhir.exceptions.TerminologyServiceException;
 import org.hl7.fhir.r5.context.ContextUtilities;
 import org.hl7.fhir.r5.context.IWorkerContext;
-import org.hl7.fhir.r5.context.IWorkerContext.ITerminologyOperationDetails;
+import org.hl7.fhir.r5.context.BaseWorkerContext.ITerminologyOperationDetails;
```

Comment:
@grahamegrieve This is an example of how ITerminologyOperationDetails is now bound to an implementation rather than and interface like IWorkerContext.